cmd/run.go

```diff
@@ -178,6 +180,7 @@ To skip bundle verification, use the --skip-verify flag.
 	runCommand.Flags().StringVarP(&cmdParams.tlsCACertFile, "tls-ca-cert-file", "", "", "set path of TLS CA cert file")
 	runCommand.Flags().VarP(cmdParams.authentication, "authentication", "", "set authentication scheme")
 	runCommand.Flags().VarP(cmdParams.authorization, "authorization", "", "set authorization scheme")
+	runCommand.Flags().VarP(cmdParams.minTLSVersion, "min-tls-version", "", "set minimum tls version to be used by opa server, default is 1.2")
```

Comment:
❓ Would it make sense to use [`StringVar`](https://pkg.go.dev/github.com/spf13/pflag#FlagSet.StringVar) here? It lets us _not_ set a short flag (which we don't use here), but instead gives us the option of stating the default right away. 🤔 I might be missing something wrt the interplay of util.NewEnumFlag and this here, though... 

Also, let's capitalize consistently:
```suggestion
	runCommand.Flags().VarP(cmdParams.minTLSVersion, "min-tls-version", "", "set minimum TLS version to be used by OPA's server, default is 1.2")
```

Reply:
@srenatus setting it as enum would allow the user to enter strings only from suggested ones and give an error when anything apart from these strings are entered.

`invalid argument "1.6" for "--min-tls-version" flag: must be one of {1.0,1.1,1.2,1.3}` 

Do we still want to go ahead with **stringVar** ?

Reply:
Thanks for the explanation 😃